File: jvm/src/main/kotlin/com/jetbrains/signatureverifier/NullLogger.kt
package com.jetbrains.signatureverifier

import org.jetbrains.annotations.NotNull

class NullLogger : ILogger {
  companion object {
    @JvmStatic
    val Instance = NullLogger()
  }

  override fun Info(@NotNull str: String) {
  }

  override fun Warning(@NotNull str: String) {
  }

  override fun Error(@NotNull str: String) {
  }

  override fun Trace(@NotNull str: String) {
  }
}

File: net/JetBrains.FormatRipper/src/NullLogger.cs
namespace JetBrains.SignatureVerifier
{
  public sealed class NullLogger : ILogger
  {
    public static readonly ILogger Instance = new NullLogger();

    private NullLogger()
    {
    }

    void ILogger.Info(string str)
    {
    }

    void ILogger.Warning(string str)
    {
    }

    void ILogger.Error(string str)
    {
    }

    void ILogger.Trace(string str)
    {
    }
  }
}